misc/test_site/mac_addrs/3c5ab41e8f0a/module_config.json

```diff
@@ -10,7 +10,7 @@
     "name": "*** Name ***",
     "serial": "*** Serial ***",
     "type": "*** Type ***",
-    "firmware_version": "*** Firmware Version ***"
+    "firmware_version": "?\\xFF\\xFF\\x19,>u\\x08\\x00no"
```

Comment:
This doesnt look correct, looks as though some bytes are being inverted or not represented correctly as ascii

Reply:
That's the output I get from bacnet4j when querying it with nmap - I believe it's meant to be 1.0.1 - still trying to figure out what encoding it's in!